src/pages/login/index.page.tsx

```diff
@@ -71,38 +76,14 @@ function Login() {
             id="password"
             label="비밀번호"
             hasLabel={true}
```

Comment:
```suggestion
            hasLabel={true}
```
어 근데 여기는 true가 있네요...?